### PR TITLE
Add Chicago file to class repo

### LINKED ISSUE #3: eabedrapo tracks progress on Github Flow
- [x] Create a branch (git)
- [x] Add some commits (git)
- [x] Open a pull request (only Github)
- [ ] Additional commits after conversation (git)
- [ ] Merge into Master (git)

### PR DESCRIPTION
**Pull request by eabedrapo**
This PR adds a Chicago file to the class repo. This PR closes #3 

I'm new in town:
- Who has been in Chicago before?
- What other restaurants would you recommend to get some nice beers?
- Which other nice attractions for tourists do you think I should add?
- What other places are locals enjoying on a daily base?

Thanks!

### CODE REVIEW

File: eabedrapo-chicago.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# Chicago, IL
+
+### Restaurants
+- Moodies
+
+### Attractions for Tourists
+- Lake Michigan
+
+### Attractions for Locals
```
Comment (wvargas10) [inline eabedrapo-chicago.md]: @eabedrapo what to do in the loop?

Comment (wvargas10) [inline eabedrapo-chicago.md]: @eabedrapo cubs or white sox fan?

**Comment by neonblueflame** [comment]
Who has been in Chicago before? No
What other restaurants would you recommend to get some nice beers? No, but I know restos with nice pizza :)
Which other nice attractions for tourists do you think I should add? Resto for chicago-style pizza?
What other places are locals enjoying on a daily base? Pizza? :D

File: eabedrapo-chicago.md (hunk 1 above)
Comment (eabedrapo) [inline eabedrapo-chicago.md]: I'll add those too! I have some friends who are Cubs fan but I also have friends who are Sox fans, so I haven't decided 😄

**Comment by eabedrapo** [comment]
@wvargas10 & @neonblueflame I added your suggestions on 8ee0e39. Thanks for your answers!

**Comment by eabedrapo** [comment]
Anyone wants to review my last commit and give a 👍, 🚢 or :shipit: ?

**Comment by wvargas10** [comment]
:ship:

**Pull request merged**